scripts/problems-test.ts

```
@@ -1,55 +1,87 @@
-  const tests: Array<{ testcase: number; input: string; pass: boolean }> = []
   for (const { testcase, input, pass } of problemInfo.tests) {
     if (testcase === "all") {
       for (const { id: testcaseId } of problemInfo.testcases) {
-        tests.push({ testcase: testcaseId, input, pass })
+        tests.push({
+          name: `(${slug}) testcase-${testcaseId}`,
+          callable: async () =>
+            runSubmissionTest(slug, testcaseId, input, pass),
+        })
+      }
+    } else if (testcase instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)